### ISSUE TITLE
[IfcConvert] center_model, center_model_geometry and model_offset parameters are not working

### ISSUE THREAD

**Issue by neumann-nico** [IfcOpenShell/IfcOpenShell#5700]
I just switched from IfcConvert v0.7 to v0.8 saw that the code for center model and model offset is disabled: https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.8.0/src/ifcconvert/IfcConvert.cpp#L942

Will this be enabled again, is there any way to support or is there a different way to use those parameters?
I use IfcConvert to convert IFCs to GLB.

**Comment by aothms** [comment]
Yes v0.8 was a massive migration. I'm personally not fully convinced of these flags given their limited flexibility and clunkiness of the command line interface. But it seems like core building blocks are actually still there https://github.com/IfcOpenShell/IfcOpenShell/blob/e70bc049150c29904e9711252ec752089c3ed828/src/ifcgeom/Iterator.h#L398-L446 Are you interested perhaps in re-enabling these and making sure they operate correctly on your data? Contributions are very welcome :)

**Comment by berry4u** [comment]
I confirm the issue, no offset is applied by extracting the geometry in GLB format.
Actually, the problem was already present in version 7.

**Comment by aothms** [comment]
Are you sure? Please double check with latest version: https://builds.ifcopenshell.org/

    model.ifc -y model.glb --model-offset 0;0;100

**Comment by neumann-nico** [comment]
The functionality was added back in the following commit: https://github.com/IfcOpenShell/IfcOpenShell/commit/4ec8ceb134a2cd86d9602672fe19eb23844e511c

Thanks so much!
